screens/LoginScreen.tsx

```diff
@@ -1,125 +1,92 @@
 import { NativeStackScreenProps } from '@react-navigation/native-stack';
 import { useState } from 'react';
-import {
-  Image,
-  ScrollView,
-  StyleSheet,
-  TouchableOpacity,
-  View,
-} from 'react-native';
+import { Alert, Image, ScrollView, TouchableOpacity, View } from 'react-native';
 import { Button, Text, TextInput, useTheme } from 'react-native-paper';
 import { SafeAreaView } from 'react-native-safe-area-context';
 import hushallet_logo from '../assets/logo/hushallet_logo.png';
 import { RootStackParamList } from '../navigators/RootStackNavigator';
+import { authStyles } from '../themes/styles';
+import { loginUser } from '../store/Auth/slice';
+import { useAppDispatch } from '../store/store';
 
-type Props = NativeStackScreenProps<RootStackParamList, 'Login'>;
+export type Props = NativeStackScreenProps<RootStackParamList, 'Login'>;
 
-const LoginScreen = ({ navigation }: Props) => {
+export const LoginScreen = ({ navigation }: Props) => {
```

Comment:
I would suggest making this change, it aligns more with best practices (I believe). Or if you want you can still have it this way, but make it the default export at least!
```suggestion
export default function LoginScreen({ navigation }: Props) {
```
Please note that you will have to update the import in RootStackNavigator back to (if you make it the default export
): `import LoginScreen from '../screens/LoginScreen';`